tower-http/src/services/fs/serve_dir.rs

```diff
@@ -83,11 +84,48 @@ use tower_service::Service;
 /// #     .expect("server error");
 /// # };
 /// ```
+/// # Axum Example
+/// ```
+/// use axum::{
+///     error_handling::HandleErrorExt,
+///     http::StatusCode,
+///     routing::{get, service_method_routing},
+///     Router,
+/// };
+/// use tower_http::services::ServeDir;
+/// 
+/// async fn hello() -> &'static str {
+///     "Hello world"
+/// }
+/// 
+/// #[tokio::main]
+/// async fn main() {
+///     // build our application with a single route
+///     let app = Router::new().route("/hello", get(hello)).fallback(
+///         service_method_routing::get(ServeDir::new("public")).handle_error(
+///             |error: std::io::Error| {
+///                 Ok::<_, std::convert::Infallible>((
+///                     StatusCode::INTERNAL_SERVER_ERROR,
+///                     format!("Unhandled internal error: {}", error),
+///                 ))
+///             },
+///         ),
+///     );
+/// 
+///     // run it with hyper on localhost:3000
+///     axum::Server::bind(&"0.0.0.0:3000".parse().unwrap())
+///         .serve(app.into_make_service())
+///         .await
+///         .unwrap();
+/// }
+/// ```
+
```

Comment:
I don't think tower-http's examples should include axum. I think its more appropriate for axum's examples to include tower-http.